File: fibonacci.py
def fibonacci(end):
    x = 0
    n = 1

    sequence = [1]

    for _ in range(end-1):
        sequence.append(x + n)

        n,x = sequence[-1], sequence[-2] # n should be equal to the current sum, and x being the one preceding it

    return sequence

print(fibonacci(10)) # Should print and return the first 10 fibonacci numbers

File: fibonacci.js
function fibonacci(end) {
    let x = 0;
    let n = 0;
    let sequence = [1];

    for (let i=1; i < end; i++) {
        sequence.push(x+n);

        n = sequence[sequence.length-1];
        x = sequence[sequence.length-2];

        // n should be equal to the current sum, and x being the one preceding it
    }

    return sequence
}

console.log(fibonacci(10)); // Should return and print the first 10 fibonacci numbers
